.github/config/build-group.json

```diff
@@ -18,6 +18,12 @@
     },
     {
       "image": "ghcr.io/$GITHUB_REPOSITORY/node:14-centos7-build"
+    },
+    {
+      "image": "ghcr.io/$GITHUB_REPOSITORY/node:16-alpine3.11-build"
```

Comment:
Were you also seeing issues on alpine 3.10, thus need to build on alpine 3.11?  According to https://github.com/nodejs/node/blob/master/BUILDING.md#supported-toolchains the required compiler version is >= 8.3 which is available with alpine 3.10.

Reply:
3.10 wan't checked. I went with lowest supported here: https://github.com/nodejs/docker-node/tree/main/16 (3.11) and started the builder image `FROM node:16-alpine3.11`. Should we try to push lower here also?

Reply:
Yes let's try with alpine 3.10 then, it satisfies the GCC requirement so hopefully just works.

Reply:
As per offline discussion all alpine builder images now use version 3.9 as baseline.

Where official images are no longer supported (16-alpine3.9, 16-alpine3.10, 14-alpine3.9) dockerfiles are copied verbatim from current supported version and FROM statement is modified. Form each file a test image was created. For alpine3.9 a build image is also created by adding the required tool chain to base dockerfile. Note that we do not FROM from own images to keep Prep an independent single step.

"All checks have passed" below refers to review workflow running with updated build mages.